src/v2/Components/ArtworkFilter/ArtworkFilters/index.tsx

```diff
@@ -21,6 +27,26 @@ interface ArtworkFiltersProps {
 // Some filters will be rendered only if there is the necessary data in aggregations (for example, ArtistsFilter)
 export const ArtworkFilters: React.FC<ArtworkFiltersProps> = props => {
   const { user, relayEnvironment } = props
+  const { match } = useRouter()
+
+  const isArtistPage = match.location.pathname.includes("/artist/")
+
+  const variant = useFeatureVariant("filters-expanded-experiment")
+
+  const { trackVariantView } = useTrackVariantView({
```

Comment:
(blocking): If we have a hook called `useFeatureVariant` we should keep the language 1:1 with other related functions -- `useTrackVariantView` -> `useTrackFeatureVariant`. 

"View" also presumes a few different things when really what we want here is something generic.